docs/cli/dotnet-publish.md

```diff
@@ -0,0 +1,66 @@
+dotnet-publish
+==============
+
+## NAME
+
+`dotnet-publish` - packs the application and all of its dependencies into a folder getting it ready for publishing
+
+## SYNOPSIS
+
+dotnet-publish [--framework]  
+    [--runtime] [--output]  
+    [--configuration]  
+    [< project >]  
+
+## DESCRIPTION
+
+`dotnet-publish` will compile the application, read through its dependencies specified in `project.json` and publish the resulting set of files to a directory. 
```

Comment:
Consider rewriting in present tense.  "dotnet publish compiles the application, reads through its dependencies..."